import-service/handlers/import-file-parser/index.ts

```diff
@@ -8,15 +9,24 @@ export const handler: S3Handler = (event, _context) => {
   console.log('Import product parser: ', event);
   try {
     const s3 = new S3();
+    const sqs = new SQS();
+    const { SQS_URL } = process.env;
 
     event.Records.forEach((record) => {
       const s3Stream = s3.getObject({
         Bucket: BUCKET_NAME,
         Key: record.s3.object.key
       }).createReadStream();
 
-      s3Stream.pipe(csv())
-        .on('data', (data) => console.log(data))
+      s3Stream.pipe(csv(['title', 'description', 'price']))
+        .on('data', (data) => {
+          sqs.sendMessage({
+            QueueUrl: SQS_URL,
+            MessageBody: JSON.stringify(data)
+          }, (err)=> {
+          if (err) throw err;
```

Comment:
This error handling looks dangerous for me, not sure how it would work and if the error will be caught anywhere...